### PR TITLE
[94X][Backport] DeepAK8 tagger integration

### PR DESCRIPTION
**Pull request by hqucms**
Backport of https://github.com/cms-sw/cmssw/pull/23768. 
Needs external from https://github.com/cms-sw/cmsdist/pull/4327.
The tagger is disabled in the MiniAOD sequence by default to preserve the event content.

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test with cms-sw/cmsdist#4327

**Comment by slava77** [comment]
@hqucms 
I think that we need to have the same changes here as in the 102X version.
Please check and update this PR.

**Comment by hqucms** [comment]
@slava77
Since DeepDoubleB is not enabled in 94X, the change in `DataFormats/Candidate/interface/CompositePtrCandidate.h` and `DataFormats/PatCandidates/interface/Jet.h` does not change the event content. Do you still prefer to revert the changes in 94X then?

**Comment by slava77** [comment]
On 9/14/18 9:54 AM, Huilin Qu wrote:
> Do you still prefer to revert the changes in 94X then?

yes.
I prefer a consistent change.

**Comment by hqucms** [comment]
@slava77 
OK! I made the same change in https://github.com/cms-sw/cmssw/pull/24506/commits/e2fcd6c41c5bc9af14d88c5db5e0c21ad6c0998d.

**Comment by slava77** [comment]
@cmsbuild please test with cms-sw/cmsdist#4327

File: PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py [hunk 1]
```diff
@@ -42,37 +42,35 @@ def applyDeepBtagging( process, postfix="" ) :
...
-                'pfDeepDoubleBJetTags:probQ',
-                'pfDeepDoubleBJetTags:probH',
-                ],
-            postfix = 'SlimmedAK8DeepDoubleB'+postfix,
-            printWarning = False
-            )
-        
-        # slimmedJetsAK8 with DeepDoubleB (remove DeepDoubleB-less)
+           process,
+           jetSource=cms.InputTag('slimmedJetsAK8NoDeepTags'),
```
Comment (slava77) [inline PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py]: here and below please keep spaces as in the other CMSSW versions (```jetSource = cms.InputTag('slimmedJetsAK8NoDeepTags'),```)

Comment (hqucms) [reply PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py]: @slava77 Fixed in https://github.com/cms-sw/cmssw/pull/24506/commits/cdd89f81800d6a4e6f7f629293ff009963dea7ca.

**Comment by slava77** [comment]
@cmsbuild please test with cms-sw/cmsdist#4327

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/24506	cdd89f8
- backport of  #23768 is correct in the context of 94X:
    - all relevant code is ported
    - the configuration parts are set to disable deepAK8 in the standard sequences
- jenkins tests pass and comparisons with the baseline show no differences

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
merge

**Pull request merged**